alarms/Telegram/telegram_stickers.py

```diff
@@ -0,0 +1,153 @@
+stickerlist = {
+    '1': 'BQADBAADJwADA6ZnAU_NNRcf64d1Ag',
```

Comment:
Lets add 'gym' and 'pokestop' to this - that way all the sticker ID's are in one place, and if people want to change to their down stickers there is only 1 file needed.